src/model/CreateSuccessResponse.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Patch API V1
+ * The core API used to integrate with Patch's service
+ *
+ * Contact: developers@usepatch.com
+ */
+
+import ApiClient from '../ApiClient';
+
+class CreateSuccessResponse {
```

Comment:
This is the response class for bulkOffset. This might become useful at some point.

Reply:
Note: I am still unclear if/where those classes get used in the SDK since they aren't imported anywhere.

Reply:
Like you've been saying, let's stick to the generator as close to OOTB as possible